exercises/alice_russell/ex02_oo_basics/src/Circle.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Circle.h"
+
+static const double PI = 3.141592653589793238463;
+
+double Circle::find_perimeter() const
```

Comment:
In Mantid you will also see methods and functions documented using [Doxygen](https://wiki.scilab.org/Doxygen%20documentation%20Examples).